src/transaction.js

```diff
@@ -7,35 +7,20 @@ const typeforce = require('typeforce')
 const types = require('./types')
 const varuint = require('varuint-bitcoin')
 
-function varSliceSize (someScript) {
+const varSliceSize = (someScript) => {
   const length = someScript.length
 
   return varuint.encodingLength(length) + length
 }
 
-function vectorSize (someVector) {
```

Comment:
this seems silly,  searching for `function vectorSize` is immensely useful compared to the above syntax, which, although constant, makes me think of this `function` as somehow variable...

Reply:
I love regex, so I'd just search `const xxxxx = \S+ => ` where xxxxx is the function I'm looking for... in fact that query is in my favorites actually...

Again, preference, and I'm not too set on it.